docs/docs/workflows/settings/README.md

```diff
@@ -46,7 +46,7 @@ If the step fails on all 8 retries and throws a final error, you'll receive [an
 
 By default, if a step fails on all 8 retries, and throws a final error, you'll receive [an error notification](/workflows/errors/) through your standard notification channel. But sometimes you need to investigate errors as soon as they happen. If you're connecting to your database, and receive an error that the DB is down, you may want to investigate that immediately.
 
-On any workflow with auto-rerun enabled, you can optionally choose to **Send notification on first error**. This is disabled by default so you don't get emails for transient errors, but you can enable for critical workflows where you want visibility into all errors.
+On any workflow with auto-retry enabled, you can optionally choose to **Send notification on first error**. This is disabled by default so you don't get emails for transient errors, but you can enable for critical workflows where you want visibility into all errors.
```

Comment:
```suggestion
On any workflow with auto-retry enabled, you can optionally choose to **Send notification on first error**. This is disabled by default, so you don't receive emails for transient errors. However, you can enable it for critical workflows where you want visibility into all errors.
```

Reply:
Changed "get" to "receive" and "but" to "however" to improve clarity and readability according to Google's style guide and Dreyer's English.